src/main/webapp/app/exercises/programming/assess/manual-result/programming-assessment-manual-result.service.ts

```diff
@@ -21,13 +20,15 @@ export class ProgrammingAssessmentManualResultService {
      * @param {submit} submit - Indicates whether submit or save is called
      */
     save(participationId: number, result: Result, submit = false): Observable<EntityResponseType> {
```

Comment:
we should add a TODO to make this consistent with the other ...assessment.service.ts files.
For now we could rename it and reorder the parameters

Reply:
```suggestion
    // TODO: make consistent with other *.assessment.service.ts files
    saveAssessment(result: Result, participationId: number, submit = false): Observable<EntityResponseType> {
```